python/cac_tripplanner/destinations/admin.py

```diff
@@ -92,5 +98,22 @@ def make_unpublished(self, request, queryset):
     make_unpublished.short_description = 'Unpublish selected events'
 
 
+class AttractionUserFlagsAdmin(admin.ModelAdmin):
+
+    list_display = ('name', 'been', 'want_to_go', 'liked', 'not_interested',)
+    readonly_fields = ('name', 'been', 'want_to_go', 'liked', 'not_interested',)
+    ordering = ('name',)
+    list_display_links = None
```

Comment:
Not strictly necessary, as `has_delete_permission` prevents it from working, but adding `actions = None` will remove the actions dropdown with the "Delete selected ..." action from the page entirely.